hypertrace-trace-enricher/hypertrace-trace-enricher-api/src/main/java/org/hypertrace/traceenricher/trace/util/ApiTraceGraph.java

```diff
@@ -272,12 +272,11 @@ private void buildApiNodes(StructuredTraceGraph graph) {
 
   private void buildApiNodeEdges(StructuredTraceGraph graph) {
     // 1. get all the exit boundary events from an api node
-    // 2. exit boundary events are the only ones which can call a different api node
-    // 3. find all the children of exit boundary events, which will be entry boundary nodes of
+    // 2. find all the children of exit boundary events, which will be entry boundary nodes of
```

Comment:
nit: `find all the children of exit boundary events` is not accurate since we check the children of entry boundary events too now

Reply:
Other case updated later in the method, this is the first case.